Makefile

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+# See LICENSE.txt for license information.
+
+## Docker Build Versions
+DOCKER_BUILD_IMAGE = golang:1.17.3
+DOCKER_BASE_IMAGE = alpine:3.14
+
+# Variables
+GO = go
+APP := elrond
+APPNAME := elrond
+ELROND_IMAGE ?= mattermost/elrond:test
+
+# Build variables
+COMMIT_HASH ?= $(shell git rev-parse HEAD)
+BUILD_DATE  ?= $(shell date +%FT%T%z)
+
+################################################################################
+
+export GO111MODULE=on
+
+all: check-style unittest
+
+.PHONY: check-style
+check-style: govet
+	@echo Checking for style guide compliance
+
+.PHONY: vet
+govet:
+	@echo Running govet
+	$(GO) vet ./...
+	@echo Govet success
+
+.PHONY: unittest
+unittest:
+	$(GO) test ./... -v -covermode=count -coverprofile=coverage.out
+
+# Build for distribution
+.PHONY: build
+build:
```

Comment:
vvvv minor: suggesting to support multiarch builds

Reply:
done thanks :)